### ISSUE TITLE
Improve update_fields handling with ForeignKey

### ISSUE THREAD

**Issue by hsmett** [romgar/django-dirtyfields#104]
Hello Hello!

When dealing with ForeignKeys, Django allows to provide the '{field_name}_id' fields name in the `update_fields` `save` parameter. So should you :)
To reproduce the error I got:
```
    tmwfk = TestModelWithForeignKey.objects.create(fkey=tm1)

    tmwfk.fkey = tm2
    tmwfk.save(update_fields=['fkey_id'])
```
My pull request is coming!
Thanks for your hard work!

**Comment by romgar** [comment]
Thanks for your issue creation and PR! I will have a look shortly!

**Comment by hsmett** [comment]
Hi @romgar, did you find time to look at it?

**Comment by romgar** [comment]
Sorry about the delay @hsmett ! I will have a look very soon!!

**Comment by romgar** [comment]
Fixed by #105.
